.golangci.yml

```diff
@@ -3,7 +3,7 @@ linters-settings:
     lines: 100
     statements: 50
   gocyclo:
-    min-complexity: 15
+    min-complexity: 20
```

Comment:
I would prefer you to add a `//nolint:gocyclo // whatever ` where you need to ignore gocyclo than doing that

Here you are losening the rule for everything and for ever

Reply:
https://github.com/goyek/goyek/pull/420/commits/da4ab1ea8addc4238821b198fd3e4e3e32b61fde